src/frontends/pytorch/src/op/adaptive_avg_pool3d.cpp

```diff
@@ -0,0 +1,38 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/opsets/opset10.hpp"
+#include "utils.hpp"
```

Comment:
utils is needed here? The same comment for other operations. Please remove redundant includes

Reply:
Same